src/app/dataset-view/additional-components/lineage-component/lineage.component.ts

```diff
@@ -15,125 +14,31 @@ import { LineageGraphNodeKind, LineageGraphNodeData, LineageNodeAccess } from ".
 export class LineageComponent extends BaseComponent implements OnInit {
     @Output() onClickNodeEmit = new EventEmitter<Node>();
 
-    public lineageGraphLink: Edge[] = [];
-    public lineageGraphNodes: Node[] = [];
-    public isAvailableLineageGraph = false;
+    public lineageGraphLinks$: Observable<Edge[]>;
+    public lineageGraphNodes$: Observable<Node[]>;
```

Comment:
Similarly, there should be a single Observable emitting whole graph

Reply:
Fixed on Observable object